### PR TITLE
update(code): add simple default format date/time function

### PR DESCRIPTION
**Pull request by allfayn**

### CODE REVIEW

**Review by fantkolja — APPROVED** [review]

File: packages/services/src/time.ts [hunk 1]
```diff
@@ -3,4 +3,20 @@ export class TimeService {
...
+    return `${TimeService.formatDate(source)} ${TimeService.formatTime(source)}`
+  }
+
+  public static formatTime(source: Date): string {
+    return `${('0' + source.getHours().toString()).slice(-2)}:${('0' + source.getMinutes().toString()).slice(-2)}:${(
+      '0' + source.getSeconds().toString()
+    ).slice(-2)}`
+  }
+
+  public static formatDate(source: Date): string {
```
Comment (Sodik) [inline packages/services/src/time.ts:17]: Why do you need it?
We have to use  "YYYY-MM-DD" instead.
This format was approved across all products

**Review by Sodik — CHANGES_REQUESTED** [review]

File: packages/services/src/time.ts (hunk 1 above)
Comment (allfayn) [reply packages/services/src/time.ts:17]: It is the same formatting without the involvement of third-party libraries and regardless of client localization.
`YYYY-MM-DD hh:mm:ss`

What function/library do you use for format date string?

Comment (Sodik) [reply packages/services/src/time.ts:17]: formatDate method returns 'DD-MM-YYYY' instead 'YYYY-MM-DD'
We use moment

Comment (allfayn) [reply packages/services/src/time.ts:17]: Thanks. Fixed format to `YYYY-MM-DD hh:mm:ss`

**Review by Sodik — APPROVED** [review]

**Pull request merged**